### PR TITLE
New package: python3-transifex-client-0.13.6

### PR DESCRIPTION
**Pull request by Cogitri**
Still needs some more testing

### CODE REVIEW

File: srcpkgs/python3-transifex-client/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'python3-transifex-client'
+pkgname=python3-transifex-client
+version=0.13.6
+revision=1
+wrksrc="transifex-client-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-urllib3 python3-requests python3-six python3-slugify"
```
Comment (maxice8) [inline srcpkgs/python3-transifex-client/template]: use version specifiers for slugify to help dissuade people from updating it.

File: srcpkgs/python3-slugify/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-slugify'
+pkgname=python3-slugify
+# Please don't update. Only required by Transifx-client,
+# doesn't work with newer versions
```
Comment (maxice8) [inline srcpkgs/python3-slugify/template]: To help, add an 'update' file that ignores all updates aboive 1.2.6

Comment (Cogitri) [reply srcpkgs/python3-slugify/template]: Right, thanks

File: srcpkgs/python3-transifex-client/template (hunk 1 above)
Comment (Cogitri) [reply srcpkgs/python3-transifex-client/template]: Fixed.

**Comment by maxice8** [comment]
All of them are missing pycompile_module

**Comment by maxice8** [comment]
and problably archs=noarch

**Pull request merged**